examples/platform/linux/CommissionerMain.cpp

```diff
@@ -187,6 +188,13 @@ CHIP_ERROR InitCommissioner(uint16_t commissionerPort, uint16_t udcListenPort)
     gCommissionerDiscoveryController.SetUserDirectedCommissioningServer(gCommissioner.GetUserDirectedCommissioningServer());
     gCommissionerDiscoveryController.SetCommissionerCallback(&gCommissionerCallback);
 
+    // re-init server's fabric table since DeviceControllerFactory::InitSystemState will add
+    // its Fabric if it is not already there.
+    ReturnErrorOnFailure(Server::GetInstance().GetFabricTable().ReInitFromStorage());
```

Comment:
Right on line 172 of this file is a violation of the new API documentation: `gCommissioner` has a `FabricInfo *` member.

Reply:
reverted this approach